examples/files/python.py

```diff
@@ -2,9 +2,19 @@
   Input Example
 '''
 
-f_input = open('filename.txt', 'r')
-file = f_input.readlines()    # this reads in all the files to create a list called "file"
-f_input.close()   # this is to close off the file when you are done reading in the data
+#CONVENTIONAL WAY OF READING FILES
+with open('filename.txt','r') as textFile:
+	file = textFile.readlines() # this reads in all the files to create a list called "file"
```

Comment:
Update this to have 4 spaces per tab (as per PEP-8 style guide), so that it's one less thing for me to do later.